test/e2e/bundle_e2e_test.go

```diff
@@ -54,7 +54,7 @@ var _ = Describe("Installing bundles with new object types", func() {
 			By("first installing the VPA CRD on cluster")
 			const (
 				sourceName = "test-catalog"
-				imageName  = "quay.io/olmtest/single-bundle-index:pdb"
+				imageName  = "quay.io/olmtest/single-bundle-index:pdb-v1"
```

Comment:
It would be awesome if we'd eventually be able to house and make directy modifications to static bundle/index images as text fixtures in-tree vs. we update the index images whenever we run into issues (e.g. k8s 1.22 API updates).

/lgtm